changelog/fragments/2944-change.yaml

```
@@ -0,0 +1,10 @@
+entries:
+  - description: >
+      Revert deprecation of the package manifests format. See
+      [#2755](https://github.com/operator-framework/operator-sdk/pull/2755) for
+      deprecation details. The package manifests format will be supported
+      indefinitely.
```

Nit:  Some people may interpret "indefinitely" to mean "forever" rather than "for an unspecified amount of time"
```suggestion
      deprecation details. The package manifests format is still officially
      supported by the operator framework.
```